### PR TITLE
Feature-/show-user-unverified-nudge

### PR DESCRIPTION
**Pull request by adetoye-dev**
Show nudge for unverified users 

image

### CODE REVIEW

File: src/components/AppLayout/index.tsx [hunk 1]
```diff
@@ -740,6 +741,12 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 				displayName={mainDisplay}
 				isVerified={isGood && !isIdentityUnverified}
 			/>
+			{id && isIdentityUnverified && network === 'polkadot' && (
+				<UnverifiedUserNudge
+					handleSetIdentityClick={handleIdentityButtonClick}
+					isIdentitySet={email ? true : false}
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:747]: this is wrong handling @adetoye-dev

**Review by KanishkaRajputd — CHANGES_REQUESTED** [review]

File: src/components/AppLayout/index.tsx (hunk 1 above)
Comment (adetoye-dev) [reply src/components/AppLayout/index.tsx:747]: Alright, can you help with more context  on the changes required 🙏

Thanks!

Comment (adetoye-dev) [reply src/components/AppLayout/index.tsx]: Updated.
Thanks, Kanishka!

File: yarn.lock [hunk 2]
```diff
@@ -4634,11 +4644,25 @@ cookie@0.5.0:
...
+  resolved "https://registry.yarnpkg.com/cookie/-/cookie-0.4.2.tgz#0e41f24de5ecf317947c82fc789e06a884824432"
+  integrity sha512-aSWTXFzaKWkvHO1Ny/s+ePFpvKsPnjc551iI41v3ny/ow6tBG5Vd+FuqGNhh1LxOmVzOlGUriIlOaokOvhaStA==
+
 cookiejar@^2.1.1:
   version "2.1.4"
   resolved "https://registry.yarnpkg.com/cookiejar/-/cookiejar-2.1.4.tgz#ee669c1fea2cf42dc31585469d193fef0d65771b"
   integrity sha512-LDx6oHrK+PhzLKJU9j5S7/Y3jM/mUHvD/DeI1WQmJn652iPC5Y4TBzC9l+5OMOXlyTTA+SmVUPm0HQUwpD5Jqw==
 
+cookies-next@^4.1.0:
+  version "4.1.0"
```
Comment (KanishkaRajputd) [inline yarn.lock:4658]: no need here

File: src/components/AppLayout/index.tsx [hunk 3]
```diff
@@ -318,6 +320,7 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 				const judgementProvided = infoCall?.some(([, judgement]): boolean => judgement.isFeePaid);
 				const isGood = info.identity?.judgements.some(([, judgement]): boolean => judgement.isKnownGood || judgement.isReasonable);
 				setIsGood(Boolean(isGood));
+				setIsIdentitySet(info.identity ? true : false);
```
Comment (KanishkaRajputd) [inline src/components/AppLayout/index.tsx:323]: change setIsIdentitySet(!!(info.identity && !info?.identity?.judgements?.length));

File: src/components/Nudges/UnverifiedUserNudge.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import React, { useState, useEffect } from 'react';
+import { Layout } from 'antd';
+import { CloseIcon } from '~src/ui-components/CustomIcons';
+import ShieldUserIcon from '~assets/icons/shield-user-icon.svg';
+import { setCookie, getCookie } from 'cookies-next';
+
+const UnverifiedUserNudge = ({ isIdentitySet, handleSetIdentityClick }: { handleSetIdentityClick: () => void; isIdentitySet: boolean }) => {
```
Comment (KanishkaRajputd) [inline src/components/Nudges/UnverifiedUserNudge.tsx:10]: no need to make it closable

File: src/components/Nudges/UnverifiedUserNudge.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+	useEffect(() => {
+		const viewedNudge = getCookie('nudge_viewed');
+
+		if (viewedNudge) {
+			setIsOpen(false);
+		}
+	}, [setIsOpen]);
+
+	function handleNudgeClose() {
```
Comment (KanishkaRajputd) [inline src/components/Nudges/UnverifiedUserNudge.tsx:21]: no need to use cookie

**Review by KanishkaRajputd — CHANGES_REQUESTED** [review]

**Pull request closed without merging**